Add cluster before/after load information to rebalance optimization proposal (#3471)
strimzi/strimzi-kafka-operator: documentation/modules/cruise-control/con-optimization-proposals.adoc

[discrete]
== Contents of optimization proposals

An optimization proposal consists of two main components, the _summary_ and the _broker load_.
The summary section provides an overview of the optimization proposal and from this you can gain an idea of the scale of the changes involved in the proposed rebalance.

> The broker load section allow you to see the impact of the proposed rebalance on each of the brokers in the cluster.

Reviewer: allows

|===

[discrete]
=== Broker load

In the broker load section, each broker is listed under its broker ID number. 

> For each broker a number of metrics (described in the table below) are listed. 

Reviewer 2: ```suggestion
Several metrics are listed for each broker.
```

Each metric is linked to an array containing three values.

> The first is the metric value before the optimization proposal is applied, the second is the expected value of the metric after the application of the proposal and the last value is the difference between the first two values (after - before).

Reviewer 2: ```suggestion
* The first is the metric value before the optimization proposal is applied
* The second is the expected value of the metric after the proposal is applied
* The last is the difference between the first two metric values (after minus before).
```

> The following table explains the properties contained in the optimization proposal's broker load section:

Reviewer 2: ```suggestion
The following table explains the properties contained in an optimization proposal's broker load section:
```

| JSON property               | Description

m| leaders                     | The number of replicas on this broker that are partition leaders.

m| replicas                    | The number of replicas on this broker.

m| cpuPercentage               | The CPU utilization as a percentage of the defined capacity.

m| diskUsedPercentage          | The disk utilization as a percentage of the defined capacity.

m| diskUsedMB                  | The absolute disk usage in megabytes.

m| networkOutRate              | The total network output rate for the broker.

m| leaderNetworkInRate         | The network input rate for all partition leader replicas on this broker.

m| followerNetworkInRate       | The network input rate for all follower replicas on this broker.

m| potentialMaxNetworkOutRate  | The hypothetical maximum network output rate that would be realized if this broker became the leader of all the replicas it currently hosts.

|======================================================================================================

.Additional resources

* xref:con-optimization-goals-{context}[] 

* xref:proc-generating-optimization-proposals-{context}[] 

> * xref:proc-approving-optimization-proposal-{context}[]         

Reviewer: Extra whitespace
